docs/dictionary/command/put.lcdoc

```diff
@@ -93,17 +102,13 @@ URL:
   <URL> type for <binary file|binary data>.
 
 
-local tResult
-<put> tData into url (file:  tFilePath)
-if tResult is not empty then
-
-    # Handle error here
-
-else
-
-    # Continue execution
-
-end if
+    local tResult
+    <put> tData into url (file: tFilePath)
+    if tResult is not empty then
```

Comment:
tResult has nothing put into it. It needs to have
`put the result into tResult`
Or, remove tResult altogether from this script to simplify it and simply use:
`if the result is not empty then`